indexing-service/src/main/java/org/apache/druid/indexing/overlord/http/OverlordResource.java

```diff
@@ -927,6 +932,23 @@ public Response apply(TaskRunner taskRunner)
           {
             if (taskRunner instanceof WorkerTaskRunner) {
               return Response.ok(((WorkerTaskRunner) taskRunner).getWorkers()).build();
+            } else if (taskRunner.isK8sTaskRunner()) {
```

Comment:
Can we add a comment here describing that k8staskrunner specific handling was needed since it doesn't have the concept of workers and a dummy worker is being returned here?

Reply:
of courser.